### DISCUSSION TITLE
Debugging in VS Code

### DISCUSSION THREAD

**Discussion by nathonius** [react-static/react-static#1481] [General] [answered]
Struggling to get the right config for webpack-dev-server here, and not seeing any existing issues or doc on how to get debugging working. I can post my current launch.json, but I'm sort of fumbling in the dark, tbh.

Probably important to note that I'm using the typescript template.

**Comment by nathonius** [comment, marked as answer]
Ack, of course I got it right after posting this. It was almost the default, but I just hadn't gotten it quite right yet:
``` json
{
  "version": "0.2.0",
  "configurations": [
    {
      "type": "chrome",
      "request": "launch",
      "name": "Launch Chrome against localhost",
      "url": "http://localhost:3000",
      "webRoot": "${workspaceFolder}",
      "sourceMapPathOverrides": {
        "webpack:///./*": "${webRoot}/*",
        "webpack:///src/*": "${webRoot}/*",
        "webpack:///*": "*",
        "webpack:///./~/*": "${webRoot}/node_modules/*",
        "meteor://💻app/*": "${webRoot}/*"
      }
    }
  ]
}
```

I'm sure some of those overrides are unnecessary (especially the meteor one), but hey, it works.

**Reply by SleeplessByte** [reply]
But now we have it documented! 👍
